Piece.cpp

```diff
@@ -12,7 +12,9 @@
 #include <cmath>
 
 
-Piece::Piece() {
```

Comment:
Does Piece no long have a default constructor?

Reply:
@sweetsbeats I added parameters to it.